### PR TITLE
Changes added due the removal of the path field from the tags table

### PR DESCRIPTION
**Pull request by nadeeshaan**
Changes added due the removal of the path field from the tags table

### CODE REVIEW

**Comment by nadeeshaan** [comment]
Testing:
This PR is related to the path field removal in the #__tags field.
In order to test the PR first need to remove the path field from the #__tags table. Then apply PR#24 and PR#25 in order to use the new nested set methods.

File: libraries/joomla/table/nested.php [hunk 1]
```diff
@@ -1359,7 +1359,6 @@ public function rebuild($parentId = null, $leftId = 0, $level = 0, $path = '')
 			->set('lft = ' . (int) $leftId)
 			->set('rgt = ' . (int) $rightId)
 			->set('level = ' . (int) $level)
-			->set('path = ' . $this->_db->quote($path))
```
Comment (gunjanpatel) [inline libraries/joomla/table/nested.php:1362]: This changes will affect all the componnet who is using `JTableNested` methods. Todo: Make sure that you maintain all exisitng joomla componnet. We definitly need inputs on this from community. You may write here https://groups.google.com/forum/#!forum/joomla-dev-cms if you need any further help. :)

**Comment by gunjanpatel** [comment]
@nadeeshaan I think this can be test also without removing field `path` I guess. If you will remove it right now I have doubt that it will break other components and you will have to fix path query in other components too.

**Comment by nadeeshaan** [comment]
Closing this as this is Merged in #30

**Pull request closed without merging**